### PR TITLE
fix orocos install

### PR DESCRIPTION
**Pull request by AchmadFathoni**
Resolve #132

### CODE REVIEW

File: orocos-kdl-python/PKGBUILD [hunk 1]
```diff
@@ -34,6 +34,7 @@ build() {
   cd "${srcdir}/${_dir}-${pkgver}/${_pkgname}"
   mkdir -p build && cd build
   cmake -DCMAKE_INSTALL_PREFIX=/usr \
+        -DCMAKE_BUILD_TYPE=Release \
```
Comment (acxz) [inline orocos-kdl-python/PKGBUILD:37]: We should not be using the Release mode see: https://wiki.archlinux.org/title/CMake_package_guidelines#CMake_can_automatically_override_the_default_compiler_optimization_flag

File: orocos-kdl/PKGBUILD [hunk 2]
```diff
@@ -19,11 +19,14 @@ _pkgname=orocos_kdl
 build() {
   cd "${srcdir}/${_dir}-${pkgver}/${_pkgname}"
 
-  cmake -DCMAKE_INSTALL_PREFIX=/usr .
+  cmake -DCMAKE_INSTALL_PREFIX=/usr \
+        -DCMAKE_BUILD_TYPE=Release \
```
Comment (acxz) [inline orocos-kdl/PKGBUILD:23]: Same as above

**Review by acxz — CHANGES_REQUESTED** [review]
Just a little nitpick, but looks good otherwise.

**Comment by AchmadFathoni** [comment]
Removed the release flag

**Review by acxz — APPROVED** [review]

**Comment by acxz** [comment]
thx for iterating!

**Pull request merged**